spec/model/associations_spec.rb

```diff
@@ -112,24 +112,39 @@
             data_key: :organization,
             default: nil,
             class_name: "Organization",
-            foreign_key: "organization_id",
-            path: "/organizations/:id"
```

Comment:
We need each association in the list of `<HerModel>.associations` to include a path. The documentation states this will default to `/{class_name}.pluralize/{id}` for `belongs_to` though I agree it should be `resource_path` or `collection_path` depending on the type of association.

Reply:
ok I guess that means extracting and generalizing the path building logic from fetch. I'll give it a try when I get a chance

Reply:
@keegangroth after further thought I'm more inclined to accept a default `path` of `nil` on `belongs_to` 👍  Long term we might want to refactor `.associations` to resemble something like [ActiveRecord::Reflection](https://api.rubyonrails.org/classes/ActiveRecord/Reflection/ClassMethods.html) 😅 

Until then, defaulting to `nil` would have the following, expected behavior on belongs_to associations: use `path` if provided, otherwise `build_request_path` would [default](https://github.com/remiprev/her/blob/fd192875777f11135ec2188a4a9b8346226b9561/lib/her/model/paths.rb#L96) to `resource_path`. We'd also need to update the [docs](https://github.com/remiprev/her/blob/master/lib/her/model/associations.rb#L119) for belongs_to (maybe remove the _defaults to..._ language)

@edtjones see any issues here?

Reply:
sounds good, I was not seeing a good way to get at the relevant information where the option defaults were being set. Will have a go at updating the docs.